src/callbacks.cc

```diff
@@ -272,7 +272,7 @@ void DeliveryReportDispatcher::Flush() {
   {
     scoped_mutex_lock lock(async_lock);
     outstanding_event_count = events.size();
-    const size_t flush_count = std::min(outstanding_event_count, 100UL);
+    const size_t flush_count = std::min<int>(outstanding_event_count, 100UL);
```

Comment:
Should that not be `std::min<size_t>` since that is what they are expecting back?

Reply:
Doh, yes, of course! Fixed and tested.